eng/Versions.props

```diff
@@ -9,6 +9,7 @@
     <PreReleaseVersionLabel>preview4</PreReleaseVersionLabel>
     <!-- Opt-in repo features -->
     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
+    <UsingToolIbcOptimization>true</UsingToolIbcOptimization>
```

Comment:
Do we want this to run in all OSes and frameworks? I thought we didn't want to run it in Unix and netfx. My change is enabling the restore step for all official build legs.

Reply:
The EnablePartialNgenOptimization further conditions where this runs.

Reply:
Then should EnablePartialNgenOptimization skip netfx? Or it doesn't matter?

Reply:
No we want to test netfx with optimization data since we'll ship netfx assets with optimization data.

Reply:
FYI, this data is all collected on netcoreapp.  I'm not sure how well this will translate to netfx.